CONTRIBUTING.md

```diff
@@ -38,10 +38,28 @@ as follows:
 
 ## Filenames and folder structure
 
-Snake case is preferred for file names. Our documentation follows pitchfork for
-folder structure. All documentation is in `/docs` except for special files like
-the contributing guide in the `/` folder. All images used in the documentation are
-place in the `/docs/data` folder.
+Please use snake case for file names. Our documentation follows pitchfork for
```

Comment:
Could you add an example?

Using snake case refers to adding an underscore(_) character in each space and the first letter of each word is written in lowercase. 
For example, 'use_underscore_and_lower_case'

Reply:
Fixed in https://github.com/RadeonOpenCompute/ROCm/commit/6b4368e01cd0065eaee9e7d903b7f2bbc066e22b